### DISCUSSION TITLE
Custom Display Issue

### DISCUSSION THREAD

**Discussion by xiaofeng900** [directus/directus#7353] [Archive]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

when i create the custom display with vue component just exactly like sample: https://docs.directus.io/guides/displays/
it is not showing on display item list of front end.
would you please guide me how i can create the custom field with vue file? Thanks!

### To Reproduce

Create the index.js, display.vue like https://docs.directus.io/guides/displays/

And built it with npx direcutus-extension build.
And I moved index.js to /extensions/displays/custom_display.
But it is not showing on front end.

### What version of Directus are you using?

^9.0.0-rc.85

### What version of Node.js are you using?

v16.3.0

### What database are you using?

mssql

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying Directus?

running locally

**Comment by rijkvanzanten** [comment]
@nickrum Any information available?

**Reply by nickrum** [reply]
The examples in the docs work fine on my end.

@xiaofeng900 are there any warnings or errors when building the extension, when starting the API server or in the browser's console when opening the App? You could try updating to the latest version. There have been a few improvements to extension loading that should make it more obvious where things are failing.
